lab-trent/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
You're way out of line with the starter code that the rest of the folks used... I actually have to read your code! 😂